server/server/views/profile.py

```diff
@@ -6,23 +15,29 @@
 
 
 @resource(collection_path='/profile', path='/profile/{profile_id}',
-          renderer='json')
+          renderer='json', cors_origins=('http://localhost:3000',))
 class UserProfile(object):
 
-    def __init__(self, request, context=None):
+    def __init__(self, request, context=None,):
         self.request = request
         self.context = context
 
     def __acl__(self):
         return [(Allow, Everyone, ALL_PERMISSIONS),
-                (Allow, self.owner, 'edit')]
+                (Allow, Authenticated, 'edit')]
```

Comment:
It means that the every authenticated user can edit every profile page. It does not have to work like that